### ISSUE TITLE
Full sequence of steps for training and inference

### ISSUE THREAD

**Issue by Tushar-N** [Evm7/ego4dlta-icvae#4]
Could you please update the `QUICKSTART.md` file with exact commands to run in order to train the models and generate predictions on the test set? 

I cannot fully run these experiments. Several issues exist:
- The default config uses an embedding file (but [this](https://github.com/Evm7/ego4dlta-icvae/issues/3) issue says it is not used)
- There is no `test_model.py` script. `test_all.py` fails for ICVAE. There are also several `predict*.py` files. Are these used?
- The H3M config file uses 6 input clips, making it incompatible with the ICVAE model that uses 8.
- Does H3M need to be finetuned from a recognition checkpoint? (`pretrained_weights/multitask_recognition.pt`)

Could you also please share training logs and pretrained checkpoints to verify if everything is set up correctly? Thanks
